civihr_default_theme/scss/pages/_hr-details.scss

```diff
@@ -65,3 +65,198 @@
     }
   }
 }
+
+.view-mydetails-personal {
+  .contact-id span {
+    @include views-label;
+  }
+
+  .pane-block {
+    border-bottom: 1px solid $gray-light;
+    margin-bottom:0px;
+  }
+
+  .views-row {
+    min-height: 80px;
+    padding-left: 100px;
+    position: relative;
+  }
+
+  .views-field-image-URL {
+    left: 0px;
+    position: absolute;
+    top: 0px;
+  }
+
+  .no-user-image {
+    background: url(/sites/all/modules/civihr-custom/civihr_employee_portal/images/profile-default.png) center no-repeat;
+    background-size: 80px 80px;
+    overflow: hidden;
+    text-indent: -1000px;
+  }
+
+  .views-field-image-URL img,
+  .no-user-image {
+    border-radius: 50%;
+    height: 80px;
+    width: 80px;
+  }
+
+  .chr_panel__footer {
+    border-top: none;
+    margin-top: 0px;
+  }
+
+  .views-field-nothing .field-content {
```

Comment:
also, you must always nest (applies everywhere else in the PR)
```scss
.views-field-nothing {
  .field-content {
```

Reply:
Doing websites for Compucorp clients a person told me exactly the opposite when reviewing  SCSS code, since then I thought you don't like nesting.

Reply:
Anyway I can change my code for CiviHR to always nest.

Reply:
Please use nesting, because in the end both will produce same result. But nesting keeps the code clean and more manageable.